op-node/flags/flags.go

```diff
@@ -305,6 +305,12 @@ var (
 		EnvVars:  prefixEnvVars("SAFEDB_PATH"),
 		Category: OperationsCategory,
 	}
+	L1ChainConfig = &cli.StringFlag{
+		Name:     "rollup.l1-chain-config",
+		Usage:    "JSON encoded chain configuration for the L1, either in the direct format or genesis.json format (i.e. embedded under the .config property). Not necessary if using Ethereum mainnet or Sepolia as an L1.",
+		EnvVars:  prefixEnvVars("ROLLUP_L1_CHAIN_CONFIG"),
+		Category: RollupCategory,
+	}
```

Comment:
Is this a path flag, or the JSON string directly? A path should be better, since usually the config would be available in a file. In that case, this can be change to a `cli.PathFlag`.

Reply:
and maybe order together with other `L1...` flags.